src/theme.js

```diff
@@ -122,19 +122,27 @@ export default (overrides = {}) => {
   const alertVariants = badgeVariants;
 
   const heights = {
-    xs: 28,
-    sm: 32,
-    md: 36,
+    xs: 24,
+    sm: 28,
+    md: 32,
     lg: 40,
-    xl: 44,
+    xl: 48,
   };
 
   const fontSizes = {
-    xs: 8,
-    sm: 10,
-    md: 12,
-    lg: 14,
-    xl: 16,
+    xs: 10,
+    sm: 12,
+    md: 14,
+    lg: 16,
+    xl: 18,
+  };
+
+  const inputHeights = {
+    xs: 32,
+    sm: 40,
+    md: 48,
```

Comment:
Curious why the heights are diverging - if I put a default input next to a default button, I'd expect them to be the same height.